pkgs/firehose/lib/src/changelog.dart

```diff
@@ -38,7 +38,7 @@ class Changelog {
   /// or a `+...` sequence after a `-` sequence.
   /// It should be enough for the user-cases we need it for
   /// in this package.
-  static final _versionRegex = RegExp(r'\d+\.\d+\.\d+(?:[\-+]\w+)?');
+  static final _versionRegex = RegExp(r'\d+\.\d+\.\d+(?:[\+-].*)?');
```

Comment:
Comment above is no longer correrct. We consume all characters after `+` or `-` on the line.

Consider changing `.*` to `\S*`, so we won't include trailing whitespace.